lib/runtime2/threepio/api/transform/stream.ts

```diff
@@ -0,0 +1,23 @@
+export type ApiStream = AsyncGenerator<ApiStreamChunk>;
+export type ApiStreamChunk = ApiStreamTextChunk | ApiStreamUsageChunk;
+
+/**
+ * The text chunk is the main chunk in the stream and contains the text generated by the model.
+ */
+export interface ApiStreamTextChunk {
+  type: 'text'
+  text: string
+}
+
+/**
+ * The usage chunk is the last chunk in the stream and contains the total number of tokens used by the request.
+ * It is only included if the `stream` option is set to `true`.
+ */
+export interface ApiStreamUsageChunk {
+  type: 'usage'
+  inputTokens: number
+  outputTokens: number
+  cacheWriteTokens?: number
+  cacheReadTokens?: number
+  totalCost?: number // openrouter
```

Comment:
```suggestion
  totalCost?: number // openrouter
```
这里加 openrouter 是干嘛的？